src/js/components/AppListFilterComponent.jsx

```diff
@@ -95,7 +95,10 @@ var AppListFilterComponent = React.createClass({
         filterText: filterText,
         activated: filterText !== "" || state.focused
       }, this.restorePreviousCursorPosition);
-      this.props.onChange(filterText);
+
+      this.props.onChange({
+        filterText: filterText
+      });
```

Comment:
what i meant is that you could have done `this.props.onChange({filterText})` but whatever